doc/reference/atom.rst

```diff
@@ -5,6 +5,8 @@
 
 .. jl:autofunction:: src/Atom.jl Atom
 
+.. jl:autofunction:: src/Atom.jl free
```

Comment:
The `free` function should not be needed to call by the users. We can still document then, but we might want to add a note stating this.